src/transforms/rename_fields.rs

```diff
@@ -44,11 +44,16 @@ impl TransformConfig for RenameFieldsConfig {
 
 impl RenameFields {
     pub fn new(fields: IndexMap<String, TomlValue>) -> crate::Result<Self> {
+        let fields =  fields
+            .into_iter()
+            .map(|kv| flatten(kv, None))
+            .collect::<crate::Result<Vec<_>>>()?;
+        let fields = fields.into_iter().flat_map(|v| v).map(|v| match v {
```

Comment:
Look at this expert double collect! 

![](https://media.giphy.com/media/KHWuDAdW6uNGAmBWZx/giphy.gif)

This is mostly so we can emit errors properly (since `Result` implements `Iterator` it kind of gets janky otherwise)